backend/app/Models/User.php

```diff
@@ -76,4 +77,10 @@ public function userTransactions()
     {
         return $this->hasMany(Transaction::class);
     }
+
+    public function sendPasswordResetNotification($token)
+    {
+        $url = '{{api}}/reset-password?token=' . $token;
+        $this->notify(new ResetPasswordNotification($url));
```

Comment:
```suggestion
        $this->notify(new ResetPasswordNotification($token));
```
remove above $uri
we should send directly the token, we dont have web for laravel